### DISCUSSION TITLE
스터디 - 4주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#44] [General]
> 4주차 스터디 목록입니다. 스터디에 주제와 갯수는 강제가 아닌 자율적으로 속도 조절해서 해주시면 됩니다.

## 스터디 일정
2021.11.25(목) 
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접, 모의면접은 상대방이 공부한 것으로 기준으로 2개 정도 질문하고 피드백 해준다.
21:35 ~ 21:50 모의 면접을 했던 상대방이 아닌 다른 사람에게도 질문하고 싶은 걸 한다.
21:50 ~ 22:00 해당 주차에 회고록 작성, 프론트엔드 질문 정리 질문 했던 댓글 남기기.

## 스터디 모음

송현 
--
- JavaScript - 화살표 함수(arrow function)
- JavaScript - array.forEach VS array.map

남은 
-- 
- JavaScript - Module system
- JavaScript - Hoisting, TDZ

두현
--
- React - Custom Hooks
- React - UnitTest

동기
--
- JavaScript - 원시 값과 객체의 비교, Ajax란?
- React - useEffect
- FrontEnd - 쿠키 - 세션 인증방식

수민
--
- React-useRef-useCallback (리액트에서 DOM 노드 접근하기)
- CORS (지난주 주제였던 CORS에서 해결방법 & 요약 추가)

영생
--
- FrontEnd - 상태관리란?
- FrontEnd - Babel(바벨)
- FrontEnd - Polyfill(폴리필)

**Comment by whdudtod1273** [comment]
영생 - 3주차 스터디 주제
- FrontEnd - 상태관리란?
- FrontEnd - Babel(바벨)
- FrontEnd - Polyfill(폴리필)

**Comment by ttumzzi** [comment]
수민
- React-useRef-useCallback (리액트에서 DOM 노드 접근하기)
- CORS (지난주 주제였던 CORS에서 해결방법 & 요약 추가)

**Comment by vi2920va** [comment]
송현
- FrontEnd- REST, RESTful 한 예시 추가
- JavaScript - 이벤트 위임, 이벤트 전파 막는 방법, 프로파 게이션 알아보기

**Comment by ki7348** [comment]
동기
* JavaScript - 원시 값과 객체의 비교, Ajax란?
* React - useEffect
* FrontEnd - 쿠키 - 세션 인증방식

**Comment by Doodream** [comment]
두현
- React - Custom Hooks
- React - UnitTest

**Reply by Doodream** [reply]
UnitTest 부분은 다음 주제로 넘길게요.

**Comment by NAMU1105** [comment]
JavaScript - Module system
JavaScript - Hoisting, TDZ

**Comment by vi2920va** [comment]
4주차 스터디 모임 소감 한마디 적어주세요

**Reply by ttumzzi** [reply]
수민 - 조금 길어지긴 했지만, 다양한 얘기를 나눠서 좋았습니다~

**Reply by ki7348** [reply]
동기 - 다양한 얘기 나눠서 좋았고 부족한 내용들 추가해서 정리하겠습니다

**Reply by Doodream** [reply]
두현 - svg가 무엇인지 키워드에 대해 얻었고 이력서 관련 Notion pdf 내보내기 기능을 알게되서 좋았습니다. 그리고 다들 따듯한 격려를 해주셔서 잘 취뽀하겠습니다. 감사합니다.

**Reply by NAMU1105** [reply]
남은 - 영생님 코테 팁이랑 면접 이야기를 들을 수 있어서 좋았습니다. 다음에는 주중에 꼭 참여해서 모든 분들과 함께 스터디 진행하면 더 좋을 것 같습니다!

**Reply by whdudtod1273** [reply]
영생 - 처음 남은 님과 스터디를 진행했는데 서로의 이야기들을 나눌 수 있어서 좋았습니다!
